constants.xml

```diff
@@ -198,6 +198,7 @@
         <BROADCAST_INTERVAL>60</BROADCAST_INTERVAL>
         <BROADCAST_EXPIRY>600</BROADCAST_EXPIRY>
         <FETCH_LOOKUP_MSG_MAX_RETRY>3</FETCH_LOOKUP_MSG_MAX_RETRY>
+        <FETCH_DS_BLOCK_LIMIT>2</FETCH_DS_BLOCK_LIMIT>
```

Comment:
Should this constant be in `seed` section ? P2PComm deals with the network layer.